test/helpers/kubectl.go

```diff
@@ -341,13 +341,13 @@ func (kub *Kubectl) Logs(namespace string, pod string) *CmdRes {
 // ManifestGet returns the full path of the given manifest corresponding to the
 // Kubernetes version being tested, if such a manifest exists, if not it
 // returns the global manifest file.
-func (kub *Kubectl) ManifestGet(manifestFilename string) string {
-	fullPath := fmt.Sprintf("%s/%s/%s", manifestsPath, GetCurrentK8SEnv(), manifestFilename)
+func ManifestGet(manifestFilename string) string {
```

Comment:
I'm agree with remove the kubectl here, but I would move this function to another place, `helpers/utils.go` I think that it's the right place.